src/pickups/datastore/pickups.js

```diff
@@ -36,8 +37,9 @@ export default {
     },
     upcoming: (state, getters) => {
```

Comment:
better rename to something like `upcomingAndRecentlyStarted`

Reply:
...or split into a separate getter `started`

Reply:
This is not really resolved, `upcoming` still contains started pickups. Would propose the rename, seems simpler at this stage (compared to having a separate getter).

Reply:
Ah right! We forgot...